packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -148,28 +151,27 @@ export const styles = css`
         ButtonAppearance.outline,
         css`
             .control {
+                box-shadow: 0px 0px 0px ${borderWidth}
+                    rgba(${actionRgbPartialColor}, 0.3) inset;
                 background-color: transparent;
```

Comment:
I wonder why we're setting `background-color: transparent` in so many places. It's already set that way in the `.control` block at the top. I'd see if you can clean some/all of these up.